README.ja.md

```
@@ -635,6 +635,8 @@ C# では同一アセンブリ内であれば、宣言されている名前空
 - SMA0080: Internal cross-namespace access
     - 別の名前空間から `internal`（および `protected internal`）の型・メンバー・メソッド・コンストラクターへのアクセスを禁止します。
     - 親名前空間や兄弟名前空間も別境界として扱います（例: `Foo.Bar` から `Foo` や `Foo.Other` で宣言されたシンボルへはアクセスできません）。
+    - **例外**: `internal` メンバーが `Core` という名前のリーフ名前空間（ハードコード）または [構成](#アナライザーの構成方法) で指定された他の名前空間内に定義されている場合、アクセスが許可されます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)